backend/VoterRegistrationService/README.md

```diff
@@ -0,0 +1,105 @@
+# AzureFunction
+Initial setup for AccessAmerica Voter Registration 
+
+To run the Azure Functions locally using VS Code:
```

Comment:
Could you please add a note that people will need to install the Azure Core Tools installed to run this locally?